desktop/env.js

```diff
@@ -83,5 +85,8 @@ debug( '========================================================================
 // Define a global 'desktop' variable that can be used in browser windows to access config and settings
 global.desktop = {
 	config: config,
-	settings: Settings
+	settings: Settings,
+	editorContextMenu: EditorContextMenu,
+	generalContextMenu: GeneralContextMenu
```

Comment:
Might be nice to have all the menus under one key (`{ contextMenus: { editor: {}, general: {} } }`) since we may add more in the future and this will help keep the global clean.